projects/media-viewer/src/lib/annotations/services/tags/tags.services.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {Observable,} from 'rxjs';
+import {TagItemModel} from '../../models/tag-item.model';
+import { v4 as uuid } from 'uuid';
+
+@Injectable()
+export class TagsServices {
+
+  public tagItems: {[id: string]: TagItemModel[]};
+
+  constructor(private http: HttpClient) {}
+
+  public getAllTags(createdBy): Observable<TagItemModel[]> {
+    const url = `/em-anno/tags/${createdBy}`;
+    return this.http.get<TagItemModel[]>(url);
```

Comment:
maybe this should go to the annotations-api service, as that's where we have all the calls to the annotations API backend